driver/src/main.rs

```diff
@@ -97,8 +100,16 @@ struct Options {
     ///     "0x7b60655Ca240AC6c76dD29c13C45BEd969Ee6F0B": "All"
     ///   }
     /// }'
-    #[structopt(long, env = "ORDERBOOK_FILTER", default_value = "{}")]
-    orderbook_filter: OrderbookFilter,
+    #[structopt(long, env = "ORDERBOOK_BLACKLIST_FILTER", default_value = "{}")]
+    blacklist_orderbook_filter: BlacklistOrderbookFilter,
+
+    /// JSON encoded object of which tokens/orders to ignore.
+    ///
+    /// For example: '{
+    ///   "tokens": [1, 2],
+    /// }'
+    #[structopt(long, env = "ORDERBOOK_WHITELIST_FILTER", default_value = "{}")]
```

Comment:
I think it would make more sense to specify a blacklist/whitelist subtype on the token in the original filter list instead of creating a new input argument.

```
enum TokenFilter {
  Whitelist(Vec<u16>),
  Blacklist(Vec<u16>),
}
```

Right now you could theoretically specify a token whitelist and a token blacklist, which doesn't really make semantic sense.

The existing orderbook_filter could look like:

```
/// For example: '{
///   "tokens": {"Whitelist": [1, 2]},
///   "users": {
///     "0x7b60655Ca240AC6c76dD29c13C45BEd969Ee6F0A": { "OrderIds": [0, 1] },
///     "0x7b60655Ca240AC6c76dD29c13C45BEd969Ee6F0B": "All"	    
///   }
```